README.md

```diff
@@ -35,6 +35,25 @@ Add the service provider to your `config/app.php` file within the `providers` ke
 ],
 // ...
 ```
+### Configuration for Local Environment
+
+If you wish to only generators on your local environment, install via composer using --dev option.
+
+```shell
+composer require reliese/laravel --dev
+```
+
+Now add the provider in app/Providers/AppServiceProvider.php file.
+
+```shell
+public function register()
+{
+    if ($this->app->environment() == 'local') {
```

Comment:
This is so that other environments like QA or production isn't able to regenerate modules. It's bad practice to make programmatic changes directly in those environment so many people will appreciate this alternative configuration. Since I won't be using it in my production environment I rather not load it.